### PR TITLE
add the source collection reqbody validate

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Comment by tigershi** [comment]
need more discuss and other task

**Review by Xiaochao8 — COMMENTED** [review]
Hasn't it been handled by tomcat?

**Comment by tigershi** [comment]
tomcat can't give a standard error response

**Review by Xiaochao8 — DISMISSED** [review]

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java [hunk 1]
```diff
@@ -91,6 +91,9 @@ public class WebConfiguration implements WebMvcConfigurer {
 	
 	@Value("${config.client.requestIds:}")
 	private String requestIdsStr; 
+	
+	@Value("${source.request.max-size:10485760}")
```
Comment (linr211) [inline g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java:95]: Is this '10485760' duplicated with the one in the class 'CollectSourceReqBodyInterceptor'? can remove it?

File: g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties [hunk 2]
```diff
@@ -32,6 +32,7 @@ server.trace.enable=false
 #collection source server
 source.cache.flag = false
 source.cache.server.url = https://localhost:8088
+source.request.max-size = 10485760
```
Comment (linr211) [inline g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties:35]: By default, it doesn't need to set the size, comment out this setting?

Comment (linr211) [inline g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties:35]: By default, it doesn't need to set the size, comment out this setting?

Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties:36]: done

Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties:36]: done

Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/resources/application-bundle.properties:36]: done

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java]: done

**Review by Xiaochao8 — APPROVED** [review]

**Review by linr211 — APPROVED** [review]

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java]: done

**Pull request merged**